frontend/src/container/MetricsApplication/MetricsApplication.factory.ts

```diff
@@ -0,0 +1,21 @@
+import { Widgets } from 'types/api/dashboard/getAll';
+import { v4 } from 'uuid';
+
+export const getWidgetQueryBuilder = (query: Widgets['query']): Widgets => ({
+	description: '',
+	id: v4(),
+	isStacked: false,
+	nullZeroValues: '',
+	opacity: '0',
+	panelTypes: 'TIME_SERIES',
+	query,
+	queryData: {
+		data: { queryData: [] },
+		error: false,
+		errorMessage: '',
+		loading: false,
+	},
+	timePreferance: 'GLOBAL_TIME',
+	title: '',
+	stepSize: 60,
```

Comment:
Why is this step size hard-coded to 60?

Reply:
2 reasons - 

1. The postman collection had it so.
2. If I create a Dashboard, and i believe we are using query builder there, the resultant query also has step set to 60.

Is it configurable? Should user have the flexibility to change it?

Reply:
The postman collection had it for demo purposes. It should be dynamic based on the selected time range. I assume there was some logic written do that.

Reply:
@srikanthccv After analysing it further, looks like the hardcoded 'stepSize' is of not being used in the final calculation. We are indeed calculating the stepSize dynamically based on start and end time in another function. 
Hence for now am removing it altogether. Had a discussion with @palashgdev also regarding the same.

Reply:
> Hence for now am removing it altogether

I am not following this. What does removing it altogether mean? Will it be part of the request payload?

Reply:
it will be part of the request payload but it's not coming from the above function. It's coming from another function called "getQueryResults.ts" which isn't part of this PR. The start and end time are also coming from that function itself. So the stepSize is being calculated dynamically only, but the above isn't responsible for that.

Reply:
Can you point me to the code where this gets added to the request payload (not where it is calculated)?

Reply:
It should be Line No 139 in "getQueryResults.ts". here's the snippet - 

	const response = await getMetricsQueryRange({
		start: parseInt(start, 10) * 1e3,
		end: parseInt(end, 10) * 1e3,
		step: getStep({ start, end, inputFormat: 'ms' }),
		variables,
		...QueryPayload,
	});

@palashgdev Please correct me if am wrong.